tests/integration/security/rbac_v1alpha1_test.go

```diff
@@ -199,53 +200,55 @@ func TestV1_GRPC(t *testing.T) {
 				With(&d, util.EchoConfig("d", ns, false, nil, g, p)).
 				BuildOrFail(t)
 
-			cases := []rbacUtil.TestCase{
-				{
-					Request: connection.Checker{
-						From: b,
-						Options: echo.CallOptions{
-							Target:   a,
-							PortName: "grpc",
-							Scheme:   scheme.GRPC,
+			for _, mtls := range []bool{true, false} {
```

Comment:
@yangminzhu FYI I'm making this change to run the same test with or without mTLS enabled, explicitly on service a. This makes the test no longer depends on the global installation outcome of `isMTLSEnabled`.

The test itself now becomes more hermetic.